src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1204,6 +1204,12 @@ private function registerTranslatorConfiguration(array $config, ContainerBuilder
 
             $translator->replaceArgument(4, $options);
         }
+
+        // Process and register the enabled locales
+        if (null === $config['enabled_locales'] && null !== $defaultLocale = $container->getParameter('kernel.default_locale')) {
```

Comment:
can be removed if https://github.com/symfony/symfony/pull/32433/files#r301112077